rasa/shared/core/domain.py

```diff
@@ -877,6 +890,39 @@ def as_dict(self) -> Dict[Text, Any]:
             KEY_E2E_ACTIONS: self.action_texts,
         }
 
+    def new_function(self, arg1, arg2, arg3, arg4):
```

Comment:
should we add all these tests to a unit test file `test_docstrings` which is exempted from the default tests, but is covered in unit tests?

Reply:
I don't think it's a good idea, it would mean that we're testing flake8/pydocstyle themselves

Reply:
I agree with Maxime 👍